sys/include/ztimer.h

```diff
@@ -474,8 +474,103 @@ ztimer_now_t _ztimer_now_extend(ztimer_clock_t *clock);
 /**
  * @brief   Get the current time from a clock
  *
- * @warning don't compare ztimer_now() values from different clocks. The
- *          clocks are almost certainly not synchronized.
+ * There are several caveats to consider when using values returned by
+ * `ztimer_now()` (or comparing those values to results of @ref ztimer_set,
+ * which are compatible unless MODULE_ZTMIER_NOW64 is in use):
+ *
+ * * A single value has no meaning of its own. Meaningful results are only ever
+ *   produced when subtracting values from each other (in the wrapping fashion
+ *   implied by the use of unsigned integers in C).
+ *
+ *   For example, even though it may be the case in some scenarios, the value
+ *   does **not** indicate time since system startup.
+ *
+ * * Only values obtained from the same clock can be compared.
+ *
+ * * Two values can only be compared when the clock has been continuously
+ *   active between the first and the second reading.
+ *
+ *   A clock is guaranteed to be active from the time any timer is set (the
+ *   first opportunity to get a "now" value from it is the return value of @ref
+ *   ztimer_set) until the time the timer's callback returns. The clock also
+ *   stays active when timers are set back-to-back (which is the case when the
+ *   first timer's callback sets the second timer), or when they overlap (which
+ *   can be known by starting the second timer and afterwards observing that
+ *   @ref ztimer_is_set or @ref ztimer_remove returns true in a low-priority
+ *   context).
+ *
+ *   In contrast, the clock is not guaranteed to be active if a timer is
+ *   removed and then a second one is started (even if the thread does not
+ *   block between these events), or when an expiring timer wakes up a thread
+ *   that then sets the second timer.
+ *
+ *   If the clock was active, then the difference between the second value and
+ *   the first is then the elapsed time in the clock's unit, **modulo 2³²
```

Comment:
for all  2³² and 2⁶⁴ i would prefer another notation  like 2^32 (2**32 may be missrendered ) `** indicating bold or italic`

Reply:
We have ² and ³ around already, both in documentation and in strings, including use for exponentiation. The `**` easily misrenders, and the `^` triggers LaTeX traumata, so I've left this one out. (doxygen does have math support to do this properly, but I don't think it's worth the time it'd need to set it up for the cases where exponentiation is around).

Reply:
> `2⁶⁴³²` << why does github do this

I don't see where github mixes the 64 and 32, what do you mean?

Reply:
![image](https://user-images.githubusercontent.com/814189/153907133-980bea7d-6b1e-46eb-9a76-395b011757cd.png)
i mean `²³¹⁰⁴⁵⁶⁷⁸⁹` somehow  123 are a different super than 0456789 (in code)

Reply:
yes doc renders fine 

these are the fonts github uses for code `ui-monospace,SFMono-Regular,SF Mono,Menlo,Consolas,Liberation Mono,monospace`; 

Liberation Mono is the choosen one on this system 

ui-monospace -> sytems monospace font (working in safari) see https://caniuse.com/extended-system-fonts
SFMono -> apples default monospace
Menlo  -> apples old default monospace
Consolas -> windows default since 8

Reply:
By the way, the `gucharmap` tool on Linux offers convenient inspection of fall-back font selection -- on my system, it shows superscript 1, 2 and 3 with Bitstream Vera Sans (on right-clicking the glyph's cell), but the others in DejaVu Sans. (Sadly, I know of no tool that allows one to input a list of font names like in CSS and then tells you where there is a [Zwiebelfisch](https://de.wikipedia.org/wiki/Zwiebelfisch_(Buchdruck)) (where I contest the articles assessment that the term does not apply here)).